packages/vital-elements/bodiless.docs.json

```diff
@@ -1,11 +1,25 @@
 {
   "VitalDesignSystem": {
     "README.md": "./doc/VitalDesignSystem.md",
+    "GettingStarted": {
+      "README.md": "./doc/GettingStarted/README.md",
+      "CoreConcepts.md": "./doc/GettingStarted/CoreConcepts.md",
+      "ComponentsAndTokens.md": "./doc/GettingStarted/ComponentsAndTokens.md",
+      "ComposingFromOutside.md": "./doc/GettingStarted/ComposingFromOutside.md",
+      "ReachingInside.md": "./doc/GettingStarted/ReachingInside.md",
+      "AnatomyOfA_Token.md": "./doc/GettingStarted/AnatomyOfAToken.md",
+      "FunctionalClasses.md": "./doc/GettingStarted/FunctionalClasses.md",
+      "WorkingWithContent.md": "./doc/GettingStarted/WorkingWithContent.md",
+      "CleanComponents.md": "./doc/GettingStarted/CleanComponents.md",
```

Comment:
I think these last 2 can be deleted (CleanComponents and YourFirstToken)

Reply:
✅ Deleted.